jest.config.js

```diff
@@ -1,5 +1,6 @@
 module.exports = {
   collectCoverage: true,
+  collectCoverageFrom: ['src/**/*\\.js'],
```

Comment:
So we do need to escape the period? 👀

Reply:
Ah, whoops. I had added the escape here to be consistent with the `testRegex` below, but the Jest docs indicate that `collectCoverageFrom` is using [micromatch](https://github.com/micromatch/micromatch) but `testRegex`is not (presumably it's a regular RegExp :shrug: ). So the escaping is needed for `testRegex`, but not here. I'll remove it!